### DISCUSSION TITLE
[Question]: Model Image count different than submited images (Frigate+)

### DISCUSSION THREAD

**Discussion by pappasaj** [blakeblackshear/frigate#11385] [Ask A Question]
### What is your question:

I'm new to Frigate+ and I've been submitting images and tagging them correctly per the docs.  I have over 100 images (it hasn't even been 24 hours so I'm working on it!), but when I look at the model, it says that my image count is 31.  Will my model over time use the additional images I've submitted and should I see that number increase?  Or do I have to create a new model to learn based on the new images I have submitted?

**Comment by NickM-27** [comment]
you must request a new model be generated, at which point all of the images you have currently submitted and verified will be used
